bechdel-test/bechdel.py

```diff
@@ -10,7 +10,7 @@
 
 ## Revenue Data
 numbers_df = pd.read_csv('revenue.csv',encoding='utf8',index_col=0)
-numbers_df['Released'] = pd.to_datetime(numbers_df['Released'],unit='D')
+numbers_df['Released'] = pd.to_datetime(numbers_df['Released'])
```

Comment:
There's a lot of weird merges happening here, I'm not too sure what's up (that's also why I protected the branch).